frontend/src/routes/candidate/register/PasswordSetPage.svelte

```diff
@@ -88,27 +89,13 @@ Page where candidates can set their password when logging to the app for the fir
 
     <PasswordValidator bind:validPassword password={password1} />
 
-    <label for="password1" class="hidden">{$_('candidate.password')}</label>
-    <input
-      type="password"
-      name="password1"
-      id="password1"
-      class="input mb-md w-full max-w-md"
-      placeholder={$_('candidateApp.setPassword.password')}
-      bind:value={password1}
-      autocomplete="new-password"
-      required />
-
-    <label for="password2" class="hidden">{$_('candidate.password')}</label>
-    <input
-      type="password"
-      name="password2"
-      id="password2"
-      class="input mb-md w-full max-w-md"
-      placeholder={$_('candidateApp.setPassword.confirmPassword')}
-      bind:value={password2}
-      autocomplete="new-password"
-      required />
+    <div class="mb-md w-full max-w-md space-y-10">
+      <label for="password1" class="hidden">{$_('candidate.password')}</label>
+      <PasswordField bind:password={password1} autoComplete="new-password" />
```

Comment:
The `PasswordField` in main has the id set to `password`. I now realize that it needs to take the id as a prop in order to allow the labels to be join the label to it.

Reply:
Actually, the `PasswordField` component could just include the label. Either way, it needs to have a unique identifier that the component user should provide